packages/core/src/transport/eventBridge.spec.ts

```diff
@@ -3,6 +3,8 @@ import { deleteEventBridgeStub, initEventBridgeStub } from '../../test/specHelpe
 import { getEventBridge, isEventBridgePresent } from './eventBridge'
 
 describe('isEventBridgePresent', () => {
+  const notAllowedWebViewHosts = ['foo.bar']
```

Comment:
Shouldn't it be
```suggestion
  const allowedWebViewHosts = ['foo.bar']
```
?